<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #565656;</string>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #565656;</string>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(0, 132, 255);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(0, 132, 255);
border-radius: 10px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="registrationButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(0, 132, 255);
	border-radius: 9px;
	color: #fff;
}

QPushButton:hover {
	background-color: rgb(0, 166, 255);
}
</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="classEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	border: 1px solid rgb(0, 132, 255);
	border-radius: 7px;
}
QSpinBox::up-button {
	border: 1px solid rgb(0, 132, 255);
	 border-radius: 0 1px 1px 0;
}
QSpinBox::down-button {
	 border: 1px solid rgb(0, 132, 255);
	 border-radius: 0 1px 1px 0;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #565656;</string>
   </property>
   <property name="text">
    <string>Номер класса</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
